pkg/util/kube_version.go

```diff
@@ -0,0 +1,46 @@
+package util
+
+import (
+	"regexp"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/pkg/errors"
+)
+
+// Utility method to extract the kube version from an EC version.
+// Given a version string like "2.4.0+k8s-1.30-rc0", it returns the kube semver version "1.30"
+func extractKubeVersion(ecVersion string) (*semver.Version, error) {
+	re := regexp.MustCompile(`\+k8s-(\d+\.\d+)`)
```

Comment:
move this to a global

Reply:
Done via - https://github.com/replicatedhq/kots/pull/5471/commits/8ba5e8fef1ea254b94b8b4cf097a88dd7b25b4b0